### PR TITLE
Modified the GPU documentation structure.

### PR DESCRIPTION
**Pull request by joerg84**
## Description
Previously the administrative part and usage part were intermixed.

## Urgency
- [ ] Blocker 
- [x] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Build content [locally](https://github.com/dcos/dcos-docs#test-local) and test for formatting/links.
- Add redirects to [dcos-website/redirect-files](https://github.com/dcos/dcos-website#managing-redirects).
- Change all affected versions (e.g. 1.7, 1.8, and 1.9).
- See the [contribution guidelines](https://github.com/dcos/dcos-docs#contributing).

### CODE REVIEW

File: 1.9/administration/gpu/aws-install.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 post_title: Configure Your AWS Cluster
 nav_title: Configure Your AWS Cloud Cluster
 feature_maturity: experimental
-menu_order: 2
```
Comment (klueska) [inline 1.9/administration/gpu/aws-install.md]: I seem to remember @joel-hamill  saying that they mark everything with 1 here, and some external tool knows how to do the reordering when it gets published.

Comment (joel-hamill) [inline 1.9/administration/gpu/aws-install.md]: the `1` is only for numbered steps. the menu items need their own manual entry.

**Comment by sascala** [comment]
@joerg84 I made some changes (fixed links, text edits, moved the doc out of the "marathon" section). If you are OK with the changes, I will merge.

**Comment by joerg84** [comment]
LGTM

**Comment by joel-hamill** [comment]
@joerg84 @sascala this directory should be named `/gpu/` not `/gpus/`

**Comment by joel-hamill** [comment]
@joerg84 @sascala another problem i have had to fix is that the enterprise documentation was not updated with these changes (e.g. topic moved from administration to usage were not deleted from enterprise docs).

**Pull request merged**